openadapt/utils.py

```diff
@@ -24,7 +24,7 @@
 
 from openadapt import common, config
 from openadapt.db import BaseModel
-from openadapt.logging import filter_log_messages
+from openadapt.logging_config import filter_log_messages
```

Comment:
I was facing this error
```terminal
PSYCHOPY Error: AttributeError: module 'logging' has no attribute 'getLogger'
```

Used this solution from stackoverflow [post](https://stackoverflow.com/questions/55174754/psychopy-error-attributeerror-module-logging-has-no-attribute-getlogger):
>If the current file you are working has name logging.py change it else it will give circular import error.